### ISSUE TITLE
Loss Function loss3 and Network N_net，LCnet

### ISSUE THREAD

**Issue by cccxhua** [huaqlili/unsupervised-light-enhance-ICLR2025#7]
Dear Authors:
    Thank you for sharing your outstanding work.  I have a couple of questions about released codes in the github:
①the loss function named loss3 in the line 100 of main.py;
②the network named N_net in the line 380 of lformer.py;
③the network named L_enhance_net in the line 395 of lformer.py;
    It appears that both the loss3 function and the N_net module play important roles in the training pipeline. However, I could not find corresponding descriptions or explanations of these components in the main paper.
    Additionally, the L_enhance_net module seems to correspond to LCnet, which the paper describes as “LCnet processes its features using a transformer and then applies global average pooling.” However, based on the released code, it appears that the implementation does not use a transformer structure in this module.
    Could you kindly provide some clarification or additional information on the design and purpose of loss3 and N_net? Understanding their role would greatly help in reproducing the results and deepening my comprehension of your method.
Thank you very much for your time and assistance!
Best regards！

**Comment by huaqlili** [comment]
Hello, thank you for your attention. The Loss3 and N_net are continued adoption of the baseline PairLIE's relevant designs. Since these designs are not our primary innovations, and ablation studies show they do not critically impact results in our settings, we omit detailed discussion in the main text. You can find introductions in PairLIE paper. Addtionally, LCNet's transformer architecture is in line 399.

Please feel free to contact us for further clarification.
